src/explore-education-statistics-frontend/src/modules/data-catalogue/DataCataloguePage.tsx

```diff
@@ -80,6 +82,13 @@ const DataCataloguePage: NextPage<Props> = ({
 
   const handlePublicationStepBack = () => {
     router.push('/data-catalogue', undefined, { shallow: true });
+    updateState(s => {
```

Comment:
Cool, good to know. I've changed this to `draft` instead of `s`. Haven't read much about immer. Assume we typically reach for it when dealing with more complex state that doesn't quite fit in something like `useState` or `useContent`?